test/simple_malloc.cc

```diff
@@ -327,6 +329,16 @@ int main(int argc, char *argv[]) {
     app.add_option("--initial-delay", opts.initial_delay, "Initial delay (ms)")
         ->default_val(0)
         ->check(CLI::NonNegativeNumber);
+    app.add_option("--nice", opts.nice, "Linux niceness setting")
+        ->default_val(19)
```

Comment:
Proposal is to set the default to be as nice as possible this way ddprof is scheduled in priority.

Reply:
I think we should not change the default niceness for simple_malloc, since it's not an "expected" behaviour. I would rather:
* explicitly change its niceness in simple_malloc-ut.sh
* or decrease niceness of ddprof in simple_malloc-ut.sh, which should have a similar effect and not require changing simple_malloc

Reply:
As ddprof is a wrapper, I don't see how you can do that from the shell part of things.

Reply:
I was proposing to either:
* by default don't change the niceness of simple_malloc and call `simple_malloc` in simple_malloc-ut.sh explicitly with `--nice 19` option (add it to `${opts}`)
*  instead of increasing the niceness of simple_malloc, decrease the niceness of ddprof with `--nice` option.